### PR TITLE
lesson-4 and lesson-5

### PR DESCRIPTION
**Pull request by YuliyaCher**
I don’t know how to align the text below the image on one line. therefore, there will definitely be errors. Thank you.

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style2.css" />
```
Comment (ghost) [inline Triphouse-project/index.html]: Seems it is wrong URL `./style2.css`.

File: Triphouse-project/style.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    width: 100px;
+    height: 4px;
+    margin: 0px auto;
+    top: 20px;
+    left: 0;
+    right: 0;
+    background: #f5bd41;
+}
+.content_item img {
+    display: flex;
```
Comment (ghost) [inline Triphouse-project/style.css]: Why is it flex ?
Add style how on image
2021-04-12_15h50_14

File: Triphouse-project/index.html (hunk 1 above)
Comment (YuliyaCher) [reply Triphouse-project/index.html]: changed the way

File: Triphouse-project/style.css (hunk 2 above)
Comment (YuliyaCher) [reply Triphouse-project/style.css]: fixed the error

**Pull request merged**